core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -651,6 +657,20 @@ private ExecutionModel parseCreateTable(GenericLexer lexer, SqlExecutionContext
         throw errUnexpected(lexer, tok);
     }
 
+    private void parseLikeTableName(GenericLexer lexer, CreateTableModel model) throws SqlException {
+        CharSequence tok;
+        tok = tok(lexer, "table name");
+        model.setLikeTableName(nextLiteral(GenericLexer.assertNoDotsAndSlashes(GenericLexer.unquote(tok), lexer.lastTokenPosition()), lexer.lastTokenPosition()));
+        tok = tok(lexer, ")");
+        if (!Chars.equals(tok, ")")) {
```

Comment:
This check is slightly more efficient:
```suggestion
        if (!Chars.equals(tok, ')')) {
```